pkg/utils/ocm.go

```diff
@@ -213,34 +213,43 @@ func loadOCMConfig() (*Config, error) {
 	return cfg, nil
 }
 
-func getOcmConfiguration() (*Config, error) {
+func getOcmConfiguration(ocmConfigLoader func() (*Config, error)) (*Config, error) {
 	tokenEnv := os.Getenv("OCM_TOKEN")
 	urlEnv := os.Getenv("OCM_URL")
 	refreshTokenEnv := os.Getenv("OCM_REFRESH_TOKEN") // Unlikely to be set, but check anyway
 
 	config := &Config{}
-	var err error
+	var (
+		defaultConfig          *Config
+		defaultConfigLoadError error
+	)
 
 	// If neither of the token ENVS 'OCM_TOKEN', 'OCM_REFRESH_TOKEN' are set, or 'OCM_URL' is not set,
 	// use the configuration file as base.
 	// We don't want to always load this, because the user might only use environment variables.
-	if (tokenEnv == "" && refreshTokenEnv == "") || urlEnv == "" {
-		config, err = loadOCMConfig()
-		if err != nil {
-			return &Config{}, fmt.Errorf("Could not load OCM configuration file")
+	if tokenEnv == "" || refreshTokenEnv == "" || urlEnv == "" {
```

Comment:
Does this warrant changing the language of the above comment?

Reply:
+1 on changing the above documentation to explain this change. 
From my understanding we only need either the refresh token or the token alongside an URL for an ocm connection, which is why we had this logic. Is this understanding wrong?

Reply:
When I had OCM_TOKEN set and not OCM_REFRESH_TOKEN, I encountered the error below. I'll comment more privately. 

```
FATA[0002] Cannot retrieve clusters: "Get \"https://api.stage.openshift.com/api/clusters_mgmt/v1/clusters?search=omitted&size=50\": can't get access token: invalid_grant: Invalid refresh token" 
```